### PR TITLE
Added logs to dummy backend initialization

### PR DESCRIPTION
**Pull request by SirePi**
PR for issue #565

### CODE REVIEW

File: Source/Core/Duality/Backend/Dummy/DummyGraphicsBackend.cs [hunk 1]
```diff
@@ -30,7 +30,10 @@ bool IDualityBackend.CheckAvailable()
 		{
 			return true;
 		}
-		void IDualityBackend.Init() { }
+		void IDualityBackend.Init()
+		{
+			Log.Core.WriteWarning("DummyGraphicsBackend initialized. This is unusual and may cause problems when someone tries to load images.");
```
Comment (ilexp) [inline Source/Core/Duality/Backend/Dummy/DummyGraphicsBackend.cs]: Loading images (as in CPU side image data) is actually what the system backend does. For the graphics backend, please change to a hint regarding interaction with graphics devices and rendering.

**Comment by SirePi** [comment]
Done

**Comment by ilexp** [comment]
Merged

**Comment by SirePi** [comment]
don't mind.. clicked wrong

**Pull request merged**